### PR TITLE
Adding cypress tests for file upload

### PR DESCRIPTION
**Pull request by clayliddell**
Additional test coverage

## QA Steps

- [ ] Certify Cypress tests are passing on this PR.

### CODE REVIEW

**Comment by clayliddell** [comment]
@dafeder It looks like `sites/default/files` permissions is preventing the CSV file from being uploaded and causing the new test to fail. Have we figured out what's causing permission issues with that directory?

**Comment by dafeder** [comment]
@clayliddell I don't think so; @dasumner was looking into it.

**Review by dafeder — CHANGES_REQUESTED** [review]
Remove baseUrl setting, and rename the remote file context

File: cypress.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "baseUrl": "http://2x.localtest.me",
```
Comment (clayliddell) [inline cypress.json]:
```suggestion
```

File: cypress/integration/11_admin_dataset_file_upload.spec.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import * as dkan from '../support/helpers/dkan'
+
+context('Admin dataset file upload', () => {
+  let baseurl = Cypress.config().baseUrl
+  beforeEach(() => {
+    cy.drupalLogin('testeditor', 'testeditor')
+  })
+
+  context('Create dataset with NetStorage upload', () => {
```
Comment (clayliddell) [inline cypress/integration/11_admin_dataset_file_upload.spec.js]:
```suggestion
  context('Create dataset with remote file', () => {
```

**Review by dafeder — APPROVED** [review]

**Pull request merged**